dm/templates/project/project.py.schema

```diff
@@ -179,12 +179,24 @@ properties:
         pattern: ^[0-9]{8,25}$
   labels:
     type: object
+    minProperties: 0
```

Comment:
I don't sure about this one, how it correlates with doc:

**Keys have a minimum length of 1 character and a maximum length of 63 characters, and cannot be empty. Values can be empty, and have a maximum length of 63 characters.**

Reply:
That correlates with:
**- Each resource can have multiple labels, up to a maximum of 64.**